README.md

```diff
@@ -31,18 +31,22 @@ Add fh-db as a dependency to your module and require it where required, like any
 Start the docker machine for the version of mongo you wish to test:
 ```bash
 docker run -d -p 27017:27017 mongo:2.4
```

Comment:
@wei-lee then I delete the other MongoDB 2.4 related things in the README.md as well.